package.json

```diff
@@ -209,15 +216,27 @@
     "murmurhash3js": "3.0.1",
     "ncp": "2.0.0",
     "nock": "8.0.0",
+    "node-sass": "3.8.0",
     "npm": "3.10.8",
+    "numeral": "1.5.3",
```

Comment:
I think numeral has caused us some issues in the past, though I suspect your use is probably fine. Curious what you're using it for.